roles/nfd_undeploy/tasks/main.yml

```diff
@@ -18,3 +18,8 @@
 
 - name: Delete the namespace of the NFD Operator
   command: oc --ignore-not-found=true delete -f "{{ nfd_operator_namespace }}"
+
+- name: Delete labels created by the NFD Operator
+  shell:
+    set -o pipefail;
+    oc get nodes -o yaml | sed -e '/^\s*nfd.node.kubernetes.io/d' -e '/^\s*feature.node.kubernetes.io/d' | oc replace -f -
```

Comment:
Instead of sed it's possible to use `jq` to be more explicit about what you're deleting:

```bash
oc get nodes -o json | jq 'del( .items[].metadata.labels."nfd.node.kubernetes.io" , .items[].metadata.labels."feature.node.kubernetes.io" )' | oc replace -f -
```